restApi/src/main/java/bisq/restApi/dto/PublicDiscussionChannelDto.java

```diff
@@ -0,0 +1,21 @@
+package bisq.restApi.dto;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.Set;
+
+
+@EqualsAndHashCode(onlyExplicitlyIncluded = true, callSuper = false)
+@Data
+@Schema(name = "PublicDiscussionChannel")
+public class PublicDiscussionChannelDto extends BaseDto<PublicDiscussionChannelDto> {
+
+    @EqualsAndHashCode.Include
+    protected String id;
+    private String channelName;
+    private String description;
+    private String channelAdminId; // TODO do we need the admin and moderater in the web api?
```

Comment:
Not sure yet if that will stay... I guess easiest is to mirror 1:1 the domain in the API dto classes.